compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -132,32 +148,33 @@ int entry(int argc, char **argv)
                       << "dataset: " << data_path << "\n"
                       << "input dtype: " << input_dtype << "\n"
                       << "output dtype: " << output_dtype << "\n";
-
+  
```

Comment:
We may need to check only one solver is used. It may be possible to use both solvers at once, but let's keep it as a future work.

Reply:
> We may need to check only one solver is used. It may be possible to use both solvers at once, but let's keep it as a future work.

ok.